lib/pbench/test/unit/common/test_utils.py

```diff
@@ -18,3 +20,47 @@ def test_md5sum():
         assert (
             hash_md5 == expected_hash_md5
         ), f"Expected MD5 '{expected_hash_md5}', got '{hash_md5}'"
+
+
+class TestCleanup:
```

Comment:
Are these purely new tests or were these moved from the tests of the `upload_api.py` module?